.github/workflows/get_ica_access_token.yml

```diff
@@ -0,0 +1,105 @@
+name: get-encrypted-ica-access-token
+
+on:
+  workflow_call:
+    inputs:
+      role_to_assume_arn:
+        required: true
+        type: string
+      secret_arn:
+        required: true
+        type: string
+      project_id:
+        required: true
+        type: string
+      aws_region:
+        required: true
+        type: string
+      branch_name:
+        required: true
+        type: string
+    secrets:
+      symmetrical_encryption_key:
```

Comment:
Where does this key come from?

Reply:
Just a random secret stored in https://github.com/umccr/cwl-ica/settings/secrets/actions.  

It's not currently possible to mask then return a value (like an access token), so instead encrypt it and then decrypt it